extensions/wikia/Email/Controller/DiscussionController.class.php

```diff
@@ -290,5 +290,69 @@ public function assertCanEmail() {
 			throw new Check( 'User is not subscribed to Discussions Upvote emails.' );
 		}
 	}
+}
 
+class DiscussionAtMentionController extends DiscussionController {
```

Comment:
maybe we should put that specific classes in separate files?

Reply:
Eh, I kind of like this as is. It's the convention used through the email controllers (so it's consistent), and I think it makes it easier to see what methods and member variables these sub-classed controllers inherit. Unless you have strong feelings about updating that, I think I'll leave like this :)